lang/en.json

```diff
@@ -111,12 +122,15 @@
     "cat.ssl.results.champion": "Champions!",
     "cat.ssl.r": "Results",
 
+    "cat.ssl.robocup23": "1st",
```

Comment:
Also update the 2023 technical challenges results
- 1st Ball Placement Technical Challenge
- 5th Chip Pass Challenge